regulations/html.py

```diff
@@ -23,24 +23,23 @@ def md2html(filename):
 
 class html():
 
-  def __init__(self, language, buildDir, pdfName, isTranslation=False, verbose=False):
+  def __init__(self, language, srcDir, buildDir, pdfName, isTranslation=False, verbose=False):
 
     print "%s Generating HTML in %s" % (("[" + language + "]").ljust(MAX_LANG_WIDTH + 2), buildDir)
     sys.stdout.flush()
 
     self.language = language
     self.isTranslation = isTranslation
-    self.docs_folder = "translations/" + language if self.isTranslation else "wca-regulations"
```

Comment:
I haven't checked, but just nervous: `self.docs_folder` was only ever used in this constructor?